app/src/test/java/org/simple/clinic/instantsearch/InstantSearchUpdateTest.kt

```diff
@@ -92,6 +92,7 @@ class InstantSearchUpdateTest {
         uuid = UUID.fromString("34eb57a9-d80a-4f43-9f89-1e2dade3de3f"),
         name = "PHC Obvious"
     )
+    val searchQuery = "P"
```

Comment:
Here the search query that is entered into the model is "Pat". Let's change it so that they are consistent.